airflow/providers/telegram/operators/telegram.py

```diff
@@ -63,9 +63,10 @@ def __init__(
         self.chat_id = chat_id
         self.token = token
         self.telegram_kwargs = telegram_kwargs or {}
+        self.text = text
```

Comment:
```suggestion
```

Reply:
i have updated the code because `text` template did not even work properly